ci/docker/docker-swarm.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+
+  mongoose-node:
+    image: ${IMAGE:-emcmongoose/mongoose-base}:${TAG:-latest}
+    command: ["--run-node"]
+    container_name: mongoose-node
+    hostname: mongoose-node
+    ports:
+      - "1099:1099"
+      - "9999:9999"
+      - "5005:5005"
```

Comment:
Can these ports be put into `.env `file as well? This also applies to ports in [compose file](ci/docker/docker-compose.yaml). I suggest that change since I assume we'd use `.env` file like sort of configuration file for the compose/swarm deployment.
Or they weren't put there intentionally?

Reply:
Docker swarm does not use this .env file.